build.sbt

```diff
@@ -1,23 +1,55 @@
 import sbtcrossproject.crossProject
 import sbtcrossproject.CrossType
 
-lazy val doobieVersion        = "0.6.0"
-lazy val fs2Version           = "2.2.2"
-lazy val geminiLocalesVersion = "0.2.0"
-lazy val gspMathVersion       = "0.1.10"
-lazy val kindProjectorVersion = "0.10.3"
-lazy val monocleVersion       = "2.0.1"
-lazy val paradiseVersion      = "2.1.1"
-lazy val flywayVersion        = "6.2.3"
-lazy val http4sVersion        = "0.21.0"
-lazy val scalaXmlVerson       = "1.2.0"
-lazy val mouseVersion         = "0.24"
+lazy val doobieVersion           = "0.8.6"
+lazy val fs2Version              = "2.2.2"
+lazy val geminiLocalesVersion    = "0.2.0"
+lazy val gspMathVersion          = "0.1.10+26-a6873e8c+20200215-1428-SNAPSHOT"
```

Comment:
Will be a regular version number when we release `gsp-math` for 2.13.

Reply:
I think this is a spurious error. I just tagged 0.1.11 so we can see if that works.